configs/networks/conf_esti.yml

```diff
@@ -1,6 +1,6 @@
 network:
   name: conf_wideresnet
   num_classes: '@{dataset.num_classes}'
-  pretrained: True # set 'True' to load pretrained model
+  pretrained: False # set 'True' to load pretrained model
```

Comment:
Do we need pretrained model for conf?

Reply:
In training pipeline, pretrain is False. In testing pipeline, pretrain is true